components/01-globals/icons/icons.config.js

```diff
@@ -103,6 +103,14 @@ module.exports = {
         class: 'icons__single-icon'
       },
       {
+        iconId: 'slider-next',
```

Comment:
I see that we have used another naming before (`angle`). I think we should keep this naming for left/prev and next/right variants.
![image](https://user-images.githubusercontent.com/12099511/32725923-4f4494e0-c877-11e7-9eba-20b19cae16fe.png)

Reply:
done